invenio_rdm_records/assets/semantic-ui/js/invenio_rdm_records/oaipmh/details/LinksTable.js

```diff
@@ -5,12 +5,16 @@
 // under the terms of the MIT License; see LICENSE file for more details.
 
 import PropTypes from "prop-types";
-import React, { useEffect, useState } from "react";
+import React, { Component } from "react";
 import { i18next } from "@translations/invenio_administration/i18next";
 import CopyButton from "./CopyButton";
 import { Table, Header, Dropdown, Grid } from "semantic-ui-react";
 import { http } from "@js/invenio_administration/src/api/config.js";
```

Comment:
http (axios config) should be moved to react-invenio-forms and imported from there